crates/oracle_document/coordinator/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "oracle_document"
+version = "0.0.1"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+name = "oracle_document"
+
+[dependencies]
+hdk = { workspace = true }
+serde = { workspace = true }
+holoom_types = { workspace = true }
+shared_utils = { workspace = true }
+oracle_document_integrity = { workspace = true }
+
+
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies] #avoids mio bug
+holochain = { workspace = true, default-features = false, features = ["test_utils"] }
+tokio = "1.38.0"
```

Comment:
sweet test deps again?

Reply:
yeah each coordinator zome has its own testing dependencies and tests directory